comp/otelcol/otlp/collector.go

```diff
@@ -58,10 +104,23 @@ func getComponents(s serializer.MetricSerializer, logsAgentChannel chan *message
 	if err != nil {
 		errs = append(errs, err)
 	}
+	addr := fmt.Sprintf("http://localhost:%s/v0.6/stats", pkgconfigsetup.Datadog.GetString("apm_config.receiver_port"))
```

Comment:
This seems like a field that should be part of the component's configuration

Reply:
done.